L10nSnapshotTests/L10nSuite2SnapshotTests.swift

```diff
@@ -120,26 +132,31 @@ class L10nSuite2SnapshotTests: L10nBaseSnapshotTests {
         app.otherElements.buttons.element(boundBy: 2).tap()
 
         let passcodeInput = springboard.secureTextFields.firstMatch
-        waitForExistence(passcodeInput, timeout: 20)
+        UIPasteboard.general.string = "foo"
+        waitForExistence(passcodeInput, timeout: 30)
         passcodeInput.tap()
-        sleep(1)
+        passcodeInput.press(forDuration: 2)
+        springboard.menuItems.firstMatch.tap()
+
         passcodeInput.typeText("foo\n")
 
         waitForExistence(app.tables["Login List"], timeout: 10)
         app.buttons.element(boundBy: 1).tap()
         waitForExistence(app.tables["Add Credential"], timeout: 10)
         snapshot("CreateLogin")
         app.tables["Add Credential"].cells.element(boundBy: 0).tap()
-        sleep(1)
-        app.keyboards.keys.element(boundBy: 3).tap()
+        waitForExistence(app.keyboards.keys.element(boundBy: 13), timeout: 5)
+        app.keyboards.keys.element(boundBy: 13).tap()
+        waitForExistence(app.tables["Add Credential"].cells.element(boundBy: 1), timeout: 15)
+
         app.tables["Add Credential"].cells.element(boundBy: 1).tap()
-        sleep(1)
-        app.keyboards.keys.element(boundBy: 3).tap()
+        tapKeyboardKey(13)
```

Comment:
What is 13? Should this be setup as a constant at the top of function w/comment?

Reply:
Yes, I could set up "13" as a constant. It seems like a key on the keyboard that input *one* character to the field so that we can save a login for the screenshot. ⌨️

Reply:
yes, that's a random key. You can use a constant and add a comment for the future so we know